dev/PersonPicture/PersonPicture.xaml

```diff
@@ -19,9 +19,26 @@
                         <VisualStateManager.VisualStateGroups>
                             <VisualStateGroup x:Name="CommonStates">
                                 <!-- Visual State when a Photo is available for display -->
-                                <VisualState x:Name="Photo" />
-                                <!-- Visual State when no Photo is available (even if no initials are available).-->
+                                <VisualState x:Name="Photo">
+                                    <VisualState.Setters>
+                                        <Setter Target="PersonPictureEllipse.Fill" Value="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=TemplateSettings.EffectiveImageBrush}"/>
+                                    </VisualState.Setters>
+                                </VisualState>
+                                <!-- Visual State when no Photo is available (but there are initials).-->
                                 <VisualState x:Name="Initials" />
+                                <VisualState x:Name="NoPhotoOrInitials">
+                                    <VisualState.Setters>
+                                        <Setter Target="InitialsTextBlock.FontFamily" Value="Segoe MDL2 Assets"/>
+                                        <Setter Target="InitialsTextBlock.Text" Value="&#xE77B;"/>
```

Comment:
Maybe a comment saying what this is would be useful.

Reply:
Added a comment.